### ISSUE TITLE
Wrong alignment in "New network" tab

### ISSUE THREAD

**Issue by diesel701** [pythops/impala#17]
I have installed Impala on my Arch linux from official repository and I have two 'view' problems:

The first one is in the "New network" tab.
The text in the columns seems aligned in the center and it cuts the name of the longest SSIDs on the left. So I see a list like this:
ationWifi1
Station2
  Wifi3
estStationWifi4

I am on a low resolution netbook (1366x768) so that could be the problem?

The other problem is that I do not see anything in the "Device" tab, but if I show info I get the data.

Thanks in advance for any type of help.

**Comment by pythops** [comment]
Can you include a screenshot ?

**Comment by pythops** [comment]
I will close this one as there is not update on it

**Comment by diesel701** [comment]
Sorry, I lost tracking of this issue. I will try to get a screen as soon I have a moment to switch on the system. thanks

**Comment by diesel701** [comment]
![1729233683_grim](https://github.com/user-attachments/assets/3de791fc-7662-4c32-97de-5f9399bd4001)
Here the screen I see.
Thanks

**Comment by diesel701** [comment]
I tested lowering the font size of the foot terminal and I can see my device, so both the problems are just a visualization resize/responsive problem

**Comment by pythops** [comment]
Yeah, I can see.  But it is not alignment, it's just that the space is not well occupied.  I will open an issue  it.  
Thanks for raising the issue
